### DISCUSSION TITLE
tutorials/client-side-graphql-react/05-codegen

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#137] [course feedback]
# tutorials/client-side-graphql-react/05-codegen

https://www.apollographql.com/tutorials/client-side-graphql-react/05-codegen

**Comment by dinugilakshika** [comment]
it is easy to undestand.

**Comment by karjac14-shine** [comment]
It did not generate `Track` `Author` and `Query` inside `/__generated__/graphql.ts`. It only worked after I downgraded `@graphql-codegen/client-preset` to version `4.2.6`

**Reply by MingXin10** [reply]
genius
